docs/fos_http_cache.md

```diff
@@ -0,0 +1,86 @@
+# FOSHttpCacheBundle usage in eZ
+
+Introduced with eZ Publish 5.4, [FOSHttpCacheBundle][fos] is supported by eZ Platform, and covers the following
+features:
+
+* Tags purging/baning (since version 1.8)
+* Http cache purge/ban
+* User context hash
+
+## Http cache clear
+Varnish proxy client from FOSHttpCache lib is now used for clearing eZ Http cache, even when using Symfony HttpCache.
```

Comment:
If "Symfony HttpCache" == "Symfony HttpCache Proxy" then no I don't think that is right afaik, at least our doc does not imply this: https://doc.ez.no/display/DEVELOPER/HTTP+Cache#HTTPCache-CachePurge